README.md

```diff
@@ -1 +1,70 @@
-# Advanced_Building_Blocks_Bubble_Sort
+![](https://img.shields.io/badge/-Ruby-rgb(199%2C%2032%2C%2039)?style=plastic&logo=ruby)
+
+# Advanced Building Blocks - Bubble Sort
+
+> The purpose of this project is to consolidate the acquired knowledge of the Ruby programming language. For this, I used the Bubble Sort programming technique.
+![screenshot](./app_screenshot.png)
+
+This web page was the first Ruby Programming Language Project of the Microverse curriculum.
+In this project, I learned the importance of the ordering of elements in Programming languages and how Ruby can help us to achieve this task.
+
+## Built With
+
+![](https://img.shields.io/badge/-Ruby-rgb(199%2C%2032%2C%2039)?style=plastic&logo=ruby)
+
+## Live Demo
+
+[Live Demo Link](https://paiza.io/projects/XbPrzvgKfjw5WpnDVxTJOQ?language=ruby/)
+
+## Getting Started
+
+This project was created with the purpose of studying the importance of variable ordering techniques in Arrays and their application in Ruby.
+
+### Prerequisites
+
+Computer running [Mac OS](https://www.apple.com/macos/big-sur/), [Windows](https://www.microsoft.com/en-us/software-download/windows10), or [Linux Ubuntu](https://ubuntu.com/download). [Ruby V.3.0.0](https://www.ruby-lang.org/es/downloads/)...
+
+### Install
+
+Getting Started
+To get a local copy up and running follow these simple example steps:
+
+- You need to [clone](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository) or [download as zip file](https://www.itprotoday.com/mobile-management-and-security/how-do-i-download-files-github) the repository on your device.
+- [Unzip](http://www.e7z.org/open-zip.htm) the repository or just clone it using git or github.
+- [Grab the index.html file and open it on your favorite browser.](https://helpdeskgeek.com/how-to/open-an-html-file-in-google-chrome/)
```

Comment:
Kindly correct this line to reflect the current project which is `bubble_sort `using the `ruby` language.